services/common/src/main/cspace/config/services/tenants/tenant-bindings-proto-unified.xml

```diff
@@ -1230,6 +1229,16 @@
 								}
 							}
 						},
+						"collectionspace_denorm:materialGroupList": {
+							"type": "object",
+							"properties": {
+								"material": {
+									"type": "keyword",
+									"copy_to": "all_field",
+									"normalizer": "sorting_normalizer"
```

Comment:
Is there a reason for using `sorting_normalizer`? I hadn't been using it for facets.